packages/record/src/consentVideo.ts

```diff
@@ -1,40 +1,71 @@
-import { JsPsych, JsPsychPlugin } from "jspsych";
-import Mustache from "mustache";
+import { LookitWindow } from "@lookit/data/dist/types";
+import Handlebars from "handlebars";
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
 import { version } from "../package.json";
-import consentVideo from "../templates/consent-video-trial.mustache";
+import consentDocTemplate from "../templates/consent-document.hbs";
+import consentVideoTrialTemplate from "../templates/consent-video-trial.hbs";
 import {
   ButtonNotFoundError,
   ImageNotFoundError,
   VideoContainerNotFoundError,
 } from "./errors";
 import Recorder from "./recorder";
-import { CSSWidthHeight } from "./types";
+import { initI18nAndTemplates } from "./utils";
+
+declare const window: LookitWindow;
 
 const info = <const>{
   name: "consent-video",
   version,
-  parameters: {},
-  data: {},
+  parameters: {
+    template: { type: ParameterType.STRING, default: "consent_005" },
+    locale: { type: ParameterType.STRING, default: "en-us" },
+    additional_video_privacy_statement: {
+      type: ParameterType.STRING,
+      default: "",
+    },
+    datause: { type: ParameterType.STRING, default: "" },
+    gdpr: { type: ParameterType.BOOL, default: false },
+    gdpr_personal_data: { type: ParameterType.STRING, default: "" },
+    gdpr_sensitive_data: { type: ParameterType.STRING, default: "" },
+    PIName: { type: ParameterType.STRING, default: "" },
+    include_databrary: { type: ParameterType.BOOL, default: false },
+    institution: { type: ParameterType.STRING, default: "" },
+    PIContact: { type: ParameterType.STRING, default: "" },
+    payment: { type: ParameterType.STRING, default: "" },
+    private_level_only: { type: ParameterType.BOOL, default: false },
+    procedures: { type: ParameterType.STRING, default: "" },
+    purpose: { type: ParameterType.STRING, default: "" },
+    research_rights_statement: { type: ParameterType.STRING, default: "" },
+    risk_statement: { type: ParameterType.STRING, default: "" },
+    voluntary_participation: { type: ParameterType.STRING, default: "" },
+    purpose_header: { type: ParameterType.STRING, default: "" },
+    procedures_header: { type: ParameterType.STRING, default: "" },
+    participation_header: { type: ParameterType.STRING, default: "" },
+    benefits_header: { type: ParameterType.STRING, default: "" },
+    risk_header: { type: ParameterType.STRING, default: "" },
+    summary_statement: { type: ParameterType.STRING, default: "" },
+    additional_segments: { type: ParameterType.COMPLEX },
```

Comment:
For this `additional_segments` parameter, are we sticking with the EFP structure, i.e. an array of objects, where each object can have properties `title` and `text`? If so, I think you can make the parameter type a little more descriptive/restrictive, like this:
```js
additional_segments: {
  type: ParameterType.COMPLEX,
  array: true,
  nested: {
    title: {
      type: ParameterType.STRING, // or HTML_STRING?
      default: '',
    // etc.
```
Here's an example from another jsPsych plugin: https://github.com/jspsych/jsPsych/blob/a411b880690b97622ced0489e439fd9819dc5a08/packages/plugin-survey-multi-select/src/index.ts#L21-L52